Core/DISET/private/InnerRPCClient.py

```diff
@@ -5,26 +5,26 @@
 
 from DIRAC.Core.DISET.private.BaseClient import BaseClient
 from DIRAC.Core.Utilities.ReturnValues import S_OK
-
+from DIRAC.Core.Utilities.DErrno import cmpError, ENOAUTH
```

Comment:
I did not see problems so far in my tests. But those were not with the extensions. However, here what is imported is from the Core DErrno, no extensions are involved ( I do not think that extensions would like to override these values )